pyrit: add page (#9230)
tldr-pages/tldr: pages/linux/pyrit.md

# pyrit

> WPA/WPA2 cracking tool using computational power.
> More information: <https://github.com/JPaulMora/Pyrit>.

- Display system cracking speed:

`pyrit benchmark`

- List available cores:

`pyrit list_cores`

> - Set ESSID:
> 
> `pyrit -e {{ESSID}} create_essid`

Reviewer: ```suggestion
- Set [e]SSID:

`pyrit -e "{{ESSID}}" create_essid`
```

Optional fix: mnemonic can be added.
Optional fix: quoting can be added if the placeholder can contain special globbing or other characters to POSIX-compatible shell.

- Read and analyze packet capture file:

> `pyrit -r {{path/to/file}} analyze`

Author: File extensions for capture files are `.cap` and `.pcap`. Need help here.

> - Read and analyze packet capture file:
> 
> `pyrit -r {{path/to/file}} analyze`

Reviewer: ```suggestion
- [r]ead and analyze a specific packet capture file:

`pyrit -r {{path/to/file.cap|path/to/file.pcap}} analyze`
```

Optional fix: mnemonic can be added.
Optional fix: two extensions can be presented via `|` syntax.

- Read and import passwords to the database:

> `pyrit -i {{path/to/file}} import_passwords`

Author: Can use `import_unique_passwords` or `unique_passwords` here. The difference is `import_unique_passwords`is faster than the `import_passwords` command. Need help here too.

> - Read and import passwords to the database:
> 
> `pyrit -i {{path/to/file}} import_passwords`

Reviewer: ```suggestion
- Read and [i]mport passwords to the current database:

`pyrit -i {{path/to/file}} {{import_unique_passwords|unique_passwords|import_passwords}}`
```

Almost the same issue as above.
Optional fix: as database placeholder is not found, it's assumed that this example referred to the current database.

> - Export passwords from database to a file:
> 
> `pyrit -o {{path/to/file}} export_passwords`

Reviewer: ```suggestion
- Exp[o]rt passwords from database to a specific file:

`pyrit -o {{path/to/file}} export_passwords`
```

Almost the same issue as above.
Optional fix: `specific` may sound better.

- Translate passwords with Pired Master Keys:

`pyrit batch`

> - Read the capture file and crack the password:
> 
> `pyrit -r {{path/to/file}} attack_db`

Reviewer: ```suggestion
- [r]ead the capture file and crack the password:

`pyrit -r {{path/to/file}} attack_db`
```

Optional fix: mnemonic can be added.
